miio/airhumidifier_mjjsq.py

```diff
@@ -0,0 +1,236 @@
+import enum
+import logging
+from collections import defaultdict
+from typing import Any, Dict
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .device import Device, DeviceException
+
+_LOGGER = logging.getLogger(__name__)
+
+MODEL_HUMIDIFIER_MJJSQ = "deerma.humidifier.mjjsq"
+
+AVAILABLE_PROPERTIES = {
+    MODEL_HUMIDIFIER_MJJSQ: [
+        "OnOff_State",
+        "TemperatureValue",
+        "Humidity_Value",
+        "HumiSet_Value",
+        "Humidifier_Gear",
+        "Led_State",
+        "TipSound_State",
+        "waterstatus",
+        "watertankstatus",
+    ]
+}
+
+
+class AirHumidifierMjjsqException(DeviceException):
+    pass
+
+
+class OperationModeMjjsq(enum.Enum):
+    Low = 1
+    Medium = 2
+    High = 3
+    Humidity = 4
+
+
+class AirHumidifierMjjsqStatus:
+    """Container for status reports from the air humidifier mjjsq."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        """
+        Response of a Air Humidifier (deerma.humidifier.mjjsq):
+
+        {'Humidifier_Gear': 4, 'Humidity_Value': 44, 'HumiSet_Value': 54,
+         'Led_State': 1, 'OnOff_State': 0, 'TemperatureValue': 21,
+         'TipSound_State': 1, 'waterstatus': 1, 'watertankstatus': 1}
+        """
+
+        self.data = data
+
+    @property
+    def power(self) -> str:
+        """Power state."""
+        return "on" if self.data["OnOff_State"] == 1 else "off"
```

Comment:
Is there a need to have both `power` and `is_on`? I'd prefer simply a single boolean variant of one of these.

Reply:
I would like to stick on this because of consistency and a nice status output of: "Power: on"

Reply:
Ok, your call.